news/_posts/2022-12-23-playtest-20221223.md

```diff
@@ -0,0 +1,116 @@
+---
+kind: article
+title: "Playtest 20221223"
+author: "PunkPun"
+created_at: 2022-12-23 18:00 +0000
+disqus_id: "playtest-20221223"
```

Comment:
We would generally keep the previous comments for posts that are tacking extra info on top of a previous one.

Reply:
makes sense